lib/everest.cpp

```diff
@@ -533,7 +539,9 @@ void Everest::provide_cmd(const std::string impl_id, const std::string cmd_name,
                 // only use validator on non-null return types
                 if (!(res_data["retval"].is_null() &&
                       (!cmd_definition.contains("result") || cmd_definition["result"].is_null()))) {
-                    json_validator validator(Config::loader, Config::format_checker);
+                    json_validator validator(
+                        [this](const json_uri& uri, json& schema) { this->config.ref_loader(uri, schema); },
+                        Config::format_checker);
```

Comment:
Yep, this validator object gets created 5 times with this "complex" construction.  This should be refactored